### PR TITLE
VY-57 -Create employee account on auth module after adding employee

### PR DESCRIPTION
**Pull request by lf-achyutpkl**

### CODE REVIEW

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

File: src/main/java/com/lftechnology/vyaguta/core/service/impl/UserAccountServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    private static final String AUTH_URL = Configuration.instance().getAuthUrl() + "users";
+
+    /*
+     * This method creates user account on auth
+     */
+    @Override
+    public User createUserAccount(Employee employee) {
+
+        User user = new User();
+        String name = employee.getFirstName() + " " + employee.getMiddleName() + " " + employee.getLastName();
```
Comment (lf-kailashraj) [inline src/main/java/com/lftechnology/vyaguta/core/service/impl/UserAccountServiceImpl.java]: think again

Comment (lf-kailashraj) [inline src/main/java/com/lftechnology/vyaguta/core/service/impl/UserAccountServiceImpl.java]: also, why not make a method fullname in Employee class?

**Pull request merged**